src/main/scala/cromwell/engine/backend/runtimeattributes/AttributeMap.scala

```diff
@@ -12,8 +12,16 @@ object AttributeMap {
   }
 }
 
-case class AttributeMap(attrs: Map[String, Seq[String]]) {
-  def get(key: RuntimeKey): Option[String] = attrs.get(key.key).flatMap(_.headOption)
+trait AttributeMapTrait {
```

Comment:
On my branch, I refactored things in such a way that `AttributeMap` wasn't even needed anymore.  If it's annoying to refactor this, perhaps I can take this on when I rebase my changes